src/webcomponents/spreadsheet/SpreadsheetRender.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState } from "react";
+import { SpreadsheetRenderType } from "../../@types/interfaces";
+
+const SpreadsheetRender: FC<SpreadsheetRenderType> = ({
+  jRef,
+  addRow,
+  downloadSheet,
+  json,
+}) => {
+  const [showJson, setShowJson] = useState(false);
+  return (
+    <div>
+      <div ref={jRef} />
+      <br />
+      <input type="button" onClick={addRow} value="Add new row" />
+      <input type="button" onClick={downloadSheet} value="Download" />
+      <input
+        type="button"
+        onClick={() => setShowJson(!showJson)}
+        value="Toggle Json After Change"
+      />
+      <div style={{ margin: "10px" }}>{showJson ? json : ""}</div>
```

Comment:
this is not the expected input format `LangugagesInput`

Reply:
there is no LangugagesInput there. What are you talking about?

Reply:
I am talking about https://github.com/iteria-poc/localization-json-jspreadsheet/blob/4c2111bfc7f8268235fb116085b21ce29b60f28a/src/%40types/interfaces.ts#L3 


src/@types/interfaces.ts 
```
export interface LanguagesInput {
```